apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nux/src/welcome-tour/utils.ts

```diff
@@ -0,0 +1,20 @@
+/**
+ * External Dependencies
+ */
+import { useMobileBreakpoint } from '@automattic/viewport-react';
+import type { Step, Steps } from '@automattic/tour-kit';
+
+export function usePrefetchTourAssets( steps: Step | Steps ): void {
+	const isMobile = useMobileBreakpoint();
+
+	if ( ! Array.isArray( steps ) ) {
+		const imgSrc = steps.meta.imgSrc;
+		new window.Image().src = isMobile && imgSrc.mobile ? imgSrc.mobile.src : imgSrc.desktop.src;
+		return;
+	}
+
+	steps.forEach( ( step: Step ) => {
+		const imgSrc = step.meta.imgSrc;
+		new window.Image().src = isMobile && imgSrc.mobile ? imgSrc.mobile.src : imgSrc.desktop.src;
+	} );
+}
```

Comment:
I think it would be a nice cleanup to also create a `hooks` folder and keep hooks contained there. So refactor this to `hooks/use-prefetch-tour-assets.ts` instead of a general-purpose utils file where anything goes.. 😬

Reply:
Done in cedcb2c